src/ftxui/component/resizable_split.cpp

```diff
@@ -264,20 +148,24 @@ class ResizableSplitBottomBase : public ComponentBase {
 /// int left_size = 10;
 /// auto left = Renderer([] { return text("Left") | center;});
 /// auto right = Renderer([] { return text("right") | center;});
-/// auto split = ResizableSplitLeft(left, right, &left_size);
+/// auto split = ResizableSplitLeft(left, right, &left_size, separatorDouble());
```

Comment:
minor: I guess can revert those changes on interface which was "polluted" by me for `ResizableSplit...` for the example code in the documentation.